### DISCUSSION TITLE
Alpine CDN broken with desktop Safari 13

### DISCUSSION THREAD

**Discussion by daronspence** [alpinejs/alpine#1540] [7. Bugs (archived)] [answered]
Open this JSbin in Safari 13 and you'll get a blank page.

https://output.jsbin.com/baratoxuwu/

Source: https://jsbin.com/baratoxuwu/edit?html,output

I'm not sure what the actual problem is as the error is pretty cryptic due to all the minification on the CDN build.

![image](https://user-images.githubusercontent.com/4062087/122477978-34054b80-cf8e-11eb-935e-4577b26c07f9.png)

**Comment by KevinBatdorf** [comment, marked as answer]
This works for me. Can anyone else confirm? What specific Safari version?

![Screen Shot 2021-06-17 at 7 24 45 PM](https://user-images.githubusercontent.com/1478421/122483966-eaece000-cffd-11eb-949b-a2a89b52fba2.png)

**Reply by daronspence** [reply]
It was 13.0.5

**Reply by KevinBatdorf** [reply]
If it's only one random Safari version it's probably not worth fixing. v13.0 doesn't even show a bleep on this breakdown: https://caniuse.com/usage-table

Safari is the modern day IE though 😭

**Reply by daronspence** [reply]
Probably. This was a colleague at work. Since I suggested Alpine to the team, I was tasked with opening a bug report. 

I’ll tell them to upgrade ;)

**Reply by calebporzio** [reply]
@Daronspence - nice, can you mark this as "answered" so Kevin gets credit for helping out? Thanks!
